src/server/game/World/World.cpp

```diff
@@ -1295,6 +1295,13 @@ void World::LoadConfigSettings(bool reload)
 
     _int_configs[CONFIG_DAILY_RBG_MIN_LEVEL_AP_REWARD] = sConfigMgr->GetOption<uint32>("DailyRBGArenaPoints.MinLevel", 71);
 
+    // Respawn
+    _float_configs[CONFIG_RESPAWN_DYNAMICRATE_CREATURE] = sConfigMgr->GetOption<float>("Respawn.DynamicRateCreature", 1.0f);
+    _int_configs[CONFIG_RESPAWN_DYNAMICMINIMUM_CREATURE] = sConfigMgr->GetOption<int32>("Respawn.DynamicMinimumCreature", 10);
```

Comment:
int32 vs uint32 type mismatch
`uint32 const timeMinimum = sWorld->getIntConfig(obj->IsGameObject() ? CONFIG_RESPAWN_DYNAMICMINIMUM_GAMEOBJECT : CONFIG_RESPAWN_DYNAMICMINIMUM_CREATURE);`

Reply:
well, it shouldn't be an int value, but we have no uint32 config containers

Reply:
example
- https://github.com/azerothcore/azerothcore-wotlk/blob/5421f13c1330af2e008ae264aabc77d09f8deae9/src/server/game/World/World.cpp#L1180

It's a minor issue, some codecheck tool might complain about this in the future :smile: